api/IDBFactory.json

```diff
@@ -0,0 +1,600 @@
+{
+  "api": {
+    "IDBFactory": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBFactory",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": [
+            {
+              "version_added": "24"
+            },
+            {
+              "version_added": "23",
+              "version_removed": "57",
+              "prefix": "webkit"
+            }
```

Comment:
Copy this information to webview_android and chrome_android.

Reply:
👍